src/messageComposer/middleware/textComposer/commands.ts

```diff
@@ -31,7 +31,7 @@ export class CommandSearchSource extends BaseSearchSource<CommandSuggestion> {
     };
   }
 
-  query(searchQuery: string) {
+  searchCommands = (searchQuery: string) => {
```

Comment:
Why do we need searchCommands method? Could we just keep `query` method?

Reply:
Yes, we can do that